examples/virtual-device-2-aws-connection/terraform.tfvars.example

```diff
@@ -0,0 +1,71 @@
+equinix_client_id      = "MyEquinixClientId"
+equinix_client_secret  = "MyEquinixSecret"
+
+#NE Acl Template
+template_name           = "test-wan-acl-template"
+template_description    = "WAN ACL template"
+template_subnet         = "172.16.25.0/24"
+template_protocol       = "TCP"
+template_src_port       = "any"
+template_dst_port       = "22"
+
+#Network Edge
+ne_name                 = "Terra_Test_router"
+ne_metro_code           = "SV"
+ne_type_code            = "C8000V"
+ne_package_code         = "network-essentials"
+ne_notifications        = ["test@eq.com"]
+ne_hostname             = "C8KV"
+ne_account_number       = "182390403"
+ne_version              = "17.11.01a"
+ne_core_count           = 2
+ne_term_length          = 1
+
+ne_ssh_key_username     = "<SSH_Key_Username>"
+ne_ssh_key_name         = "<SSH_Key_Name"
+
+#network_ssh_key
+network_public_key_name = "<Public_SSH_Key_Name>"
```

Comment:
Where are the `network_` ssh key variables used? I don't see any reference to them.

Reply:
These look like they can be removed.

Reply:
Addressed with [this](https://github.com/equinix/terraform-equinix-fabric/pull/121/commits/d7d7aae803c7e9060adc824ea6fd9ad391b36d51) commit